examples/deployment/README.md

```diff
@@ -0,0 +1,62 @@
+Deploying Trillian
+==================
+
+Want to deploy/use the Trillian General Transparency project?  Here are some common ways of getting off the ground.
+
+## Setup
+
+**Build**
+
+First, we build Docker images from source.  We'll use these to run and interact with Trillian:
+
+```shell
+# Clone the latest trillian
+git clone git@github.com:google/trillian.git
+cd trillian
+# Build Docker Images
+docker build -t trillian-server    -f examples/deployment/docker/log_server/Dockerfile .
+docker build -t trillian-signer    -f examples/deployment/docker/log_signer/Dockerfile .
+docker build -t trillian-db-client -f examples/deployment/docker/db_client/Dockerfile .
+```
+
+## Run on a Single Host
+
+For local development or simple deployments with Docker, start here:
+
+**Run With Docker Compose**
+
+After building the above containers, run the following in the director docker-compose.yml file
+
+```shell
+export DB_PASSWORD=password
+docker-compose -f examples/deployment/docker-compose.yml up
+# Tests
+curl localhost:8091/metrics
+```
+
+## Run in AWS
+
+To use AWS's managed databases, start here:
+
+```shell
+# Create Resources
+terraform plan aws/
+terraform apply aws/
+
+# Set Variables
+HOST=...
```

Comment:
merged this with automated launch userdata within terraform (more automation!) and changed to `DB_HOST`.